go.mod

```diff
@@ -98,7 +98,7 @@ replace (
 	k8s.io/csi-translation-lib v0.0.0 => k8s.io/csi-translation-lib v0.0.0-20190718190424-bef8d46b95de
 	k8s.io/gengo v0.0.0 => k8s.io/gengo v0.0.0-20190327210449-e17681d19d3a // indirect
 	k8s.io/heapster => k8s.io/heapster v1.2.0-beta.1 // indirect
-	k8s.io/klog => k8s.io/klog v0.4.0 // indirect
+	k8s.io/klog => k8s.io/klog/v2 v2.2.0
```

Comment:
v2.4.0?

Reply:
@fisherxu flow as k8s 1.19.3.
https://github.com/kubernetes/kubernetes/blob/1e11e4a2108024935ecfcb2912226cedeafd99df/go.mod#L477

Reply:
Others in this pr are v2.4.0..

Reply:
@fisherxu `go mod tidy` auto upgrade require version to 2.4.0 when I set it to 2.2.0. Let me figure out why....

Reply:
@fisherxu done, now both require and replace are v2.2.0, it is because I did not limited beehive and viaduct to v2.2.0 before.

Reply:
looks good